docker-tests/docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+
+version: "3.7"
+
+services:
+
+  back-end:
+    build: 
+      context: ../server/
+      dockerfile: ../server/Dockerfile.test
+    depends_on:
+      - database
+    ports:
+    - "3000:3000"
+    container_name: backend-server

```

Comment:
I run into a naming conflict when I've also used the Docker Compose at the root of the project. This should probably be renamed.

Reply:
Fixed